examples/private-library/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "github.com/flashbots/suapp-examples/framework"
+
+func main() {
+	privateLibrary, _ := framework.ReadArtifact("private-library.sol/PrivateLibrary.json")
+
+	fr := framework.New()
+	fr.DeployContract("private-library.sol/PublicSuapp.json").
+		SendTransaction("example", nil, privateLibrary.Code)
```

Comment:
Should we do some type of check to verify this worked?

Reply:
The check is implicit inside the framework.

Reply:
a comment to that effect would be nice here in the code

Reply:
Added comment to the function in `framework`. I will address this better in a followup PR making it a bit more clear from the example standpoint whether it expects success or not.